packages/web/src/pages/upload-page/fields/StemFilesView.tsx

```diff
@@ -31,6 +33,64 @@ const messages = {
   stemTypeDescription: 'Please select a stem type for each of your files.'
 }
 
+const useStemFileInfos = (stems: StemUploadWithFile[]) => {
```

Comment:
use /inspect endpoint to populate existing stem file sizes and types

Reply:
whoa cool, should we pull out to its own file somewhere?

Reply:
for now i think here is fine, since it's the stem files view